src/App.tsx

```diff
@@ -10,33 +12,26 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+
+  const onTabSelected = (tab: Tab) => {
+    setSelectedTabId(tab.id);
+  };
+
+  const isActive = tabs.find(tab => selectedTabId === tab.id);
```

Comment:
you don't need to make this search if you put in your state the entire tab not only tabId